README.rst

```diff
@@ -32,14 +28,11 @@ We use `poetry <https://python-poetry.org/>`_ for dependency management and run
 Once both of these are installed run this in the root folder to install all the dependencies that you need:
 :code:`poetry install`
 
-.. note::
-   If you are contributing to the project we recommend that you also run
+Note: If you are contributing to the project we recommend that you also run
```

Comment:
This brakes the note formatting. What's the reason for this?

Reply:
Notes don't render in github and just look wrong, so it dosen't 'break' it, more that it's now obviously a note in all places.